lib/utils/messages.ts

```diff
@@ -45,7 +45,7 @@ export const balenaAsciiArt = `\
 `;
 
 export const registrySecretsHelp = `\
-REGISTRY SECRETS  
+REGISTRY SECRETS
```

Comment:
The double space at the end of a line means a line break in markdown syntax, which gets converted to `<br>` (or similar) in HTML on this page: https://www.balena.io/docs/reference/balena-cli/#push-applicationordevice

Without the double space, it looks a bit broken when converted to HTML, a bit like:
```
REGISTRY SECRETS The --registry-secrets option specifies ...
```
Similar issue when viewed with the GitHub preview for files like README.md.

Reply:
That's strange - I didn't knowingly remove that.